geonode_mapstore_client/client/js/epics/gnsave.js

```diff
@@ -84,7 +84,7 @@ function parseMapBody(body, map) {
     };
 }
 
-const SaveAPI = {
+export const SaveAPI = {
```

Comment:
please remove export,
epic file should only export epics functions

```suggestion
const SaveAPI = {
```